conversions/bytes_to_int.py

```diff
@@ -0,0 +1,57 @@
+"""
+This script first converts a bytearray,
+to its corresponding signed/unsigned binary number,
+and then converts the binary number to an integer.
+
+>>> bytes_to_int(b'\\x00\\x10',False)
+16
+>>> bytes_to_int(b'\\xfc\\x00',True)
+-1024
+>>> bytes_to_int(b'\\x00\\x01',False)
+1
+>>> bytes_to_int('abc',False)
+ERROR - 'str' object cannot be interpreted as an integer
+0
+"""
+
+
+def bytes_to_int(bytes_var: bytearray, signed: bool) -> (int):
+    try:
```

Comment:
Please move the comments and tests into the function so that if someone copies the function, the documentation and tests come along with it.

The double backslashes seem unnecessary.  See https://docs.python.org/3/library/stdtypes.html#int.from_bytes Both the input and output of this function should match `int.from_bytes(_, "little")` or `int.from_bytes(_, "big")`

Reply:
I've added a byte order parameter. 

I think the double backslashes are necessary as the tests are being typed as part of a string and thus, the single backslash needs an escape sequence, which is a double backlash. Feel free to provide a better solution and correct me if I'm wrong.